prog/dftb+/lib_dftbplus/parser.F90

```diff
@@ -237,25 +233,40 @@ subroutine readParserOptions(node, root, flags)
     type(fnode), pointer :: root
 
     !> Contains parser flags on exit.
-    type(TParserFlags), intent(out) :: flags
+    type(TParserFlags), intent(inout) :: flags
 
-    integer :: inputVersion
     type(fnode), pointer :: child
+    real(dp) :: rTmp, rVersion
+
+    ! Parser input version in form x.y with year and release number, matching code (after version 8)
+    call getChild(node, "ParserVersion", child, requested=.false.)
+    if (associated(child)) then
+      call getChildValue(child, "", rTmp, rVersion)
+      flags%version%major = nint(rTmp)
+      flags%version%minor = nint(10.0_dp*(rTmp - flags%version%major))
```

Comment:
Really? A real value for the parser version? I would advocate for a version *string* here.